starter/source/output/th/hm_read_thgrki.F

```diff
@@ -241,3 +241,332 @@ SUBROUTINE HM_READ_THGRKI(
         ENDIF
         RETURN
       END
+
+
+
+            SUBROUTINE HM_READ_THGRKI_ALL(
```

Comment:
Hi

It is recommended to have 1 file per routine
( all new files should be f90)